pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1003,6 +1003,11 @@ func (f *frameworkImpl) RunPreBindPlugins(ctx context.Context, state *framework.
 	for _, pl := range f.preBindPlugins {
 		status = f.runPreBindPlugin(ctx, pl, state, pod, nodeName)
 		if !status.IsSuccess() {
+			if status.IsUnschedulable() {
+				klog.V(4).InfoS("Pod rejected by PreBind plugin", "pod", klog.KObj(pod), "plugin", pl.Name(), "status", status.Message())
```

Comment:
why 4? is there precedent?

Reply:
Yes, see https://github.com/kubernetes/kubernetes/blob/863a462eeccca4c24bf4c4e9012f5ebccade70a2/pkg/scheduler/framework/runtime/framework.go#L1150-L1154

Reply:
"Pod rejected by PreBind plugin" looks weird in the context of `prebind`, this is should be different with `permit` pulgin.

How about "Pod is unschedulable with node...."  or something similar? also pls pass in the node name to reveal more information.

Reply:
> "Pod rejected by PreBind plugin" looks weird in the context of `prebind`, this is should be different with `permit` pulgin.
> 
I think the word is appropriate, see annotation of PreBind: https://github.com/kubernetes/kubernetes/blob/bcea98234f0fdc5ce4e976758977eec72b13bfd4/pkg/scheduler/framework/interface.go#L461-L463


> also pls pass in the node name to reveal more information.

Make sense.